CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ option(SOLC_LINK_STATIC "Link solc executable statically on supported platforms"
 option(SOLC_STATIC_STDLIBS "Link solc against static versions of libgcc and libstdc++ on supported platforms" OFF)
 option(STRICT_Z3_VERSION "Require the exact version of Z3 solver expected by our test suite." ON)
 option(PEDANTIC "Enable extra warnings and pedantic build flags. Treat all warnings as errors." ON)
-option(PROFILE_OPTIMIZER_STEPS "Output performance metrics for the optimiser steps." OFF)
+option(PROFILE_OPTIMIZER_STEPS "Output performance metrics for the optimiser steps." ON)
```

Comment:
I temporarily enabled the profiler by default to make sure it does compile on all platforms. I will disable this again before marking it as ready for review.